.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @siriak @imp2002
```

@imp2002 is not a maintainer in this repository yet. You can add them here once they are a maintainer.
```suggestion
* @siriak
```